src/entity/units/bloodGolem.ts

```diff
@@ -16,15 +16,15 @@ const unit: UnitSource = {
   },
   unitProps: {
     damage: 40,
-    staminaMax: config.UNIT_BASE_STAMINA * 1.2,
-    healthMax: 60,
+    staminaMax: config.UNIT_BASE_STAMINA,
+    healthMax: 80,
```

Comment:
Keep in mind, the stamina of enemies relative to the player's cast range significantly determines their threat.  If the player has to step into danger in order to cast on them, that's a big part of the player's calculus.